README.md

```diff
@@ -30,6 +30,129 @@ services.config(({app, services}) => {
 services.start();
 ```
 
+### Using as CLI with lb Architecture
+`@labshare/services` is a package for loading APIs. These APIs can be single on multiple mounted over a single thread.
+This package supports 2 modes:
+- services: LabShare API Services Architecture.
+- lb: LabShare API LoopBack Support Architecture.
+
+This can be defined at the .labsharerc file ( if it is not present, it will use services mode).
+
+```sh
+{
+  "mode": "lb", // or services
+  "apis":[{"name":"api"}]
+}
+
+```
+
+#### Services
+LabShare API Services Architecture can load both API modes, you will need to add the config.json configuration for services APIs and the config folder for lb APIs.
+The settings for starting the server will be taken from  the config.js configuration section `services.listen`
+The APIs that need to be loaded should be defined as package dependencies at the package.json
+  "packageDependencies": [
+    "@labshare/services",
+    "@labshare/facility"
+  ],
+Now you can start the app as lsc services start.
+#### lb
+LabShare API LoopBack Support Architecture will only load lb API mode.
+In order to use it you will need to have the following configuration:
+npm i @labshare/services  at the host project
+add a .labsharerc file with the following configuration
+```sh
+{
+  "mode": "lb",
+  "apis":[{"name":"api" , packare:"@labshare/facility-api" , configAlias:"facility"}]
```

Comment:
Looks like a typo in "package". What does "configAlias" do?

Reply:
if the name of your api is different from the configuration value you can override it with this setting